cmd/apps-proxy/main.go

```diff
@@ -91,18 +91,14 @@ func run(ctx context.Context, cfg config.Config, _ []string) error {
 	oautproxylogger.SetErrOutput(loggerWriter)
 
 	// Create dependencies
-	scope, err := dependencies.NewServiceScope(ctx, cfg, proc, logger, tel, os.Stdout, os.Stderr) // nolint:forbidigo
+	d, err := dependencies.NewServiceScope(ctx, cfg, proc, logger, tel, os.Stdout, os.Stderr) // nolint:forbidigo
```

Comment:
```suggestion
	ss, err := dependencies.NewServiceScope(ctx, cfg, proc, logger, tel, os.Stdout, os.Stderr) // nolint:forbidigo
```
This looks more like service scope then `d`

Reply:
There is our convenction to use `d`, as `dependencies`, the scopes (dependency containers) have different names, we can discuss it more.